modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -19,6 +19,8 @@ Another way to publish these type of assets is using the https://anypoint.muleso
 +
 For example: `exchange asset upload --classifier raml --apiVersion v1 --name HelloWorld --mainFile helloworld.raml helloword/1.0.0 /Users/nmouso/Downloads/helloworld.raml.zip`
 * API Groups are published to Exchange from Anypoint API Manager.
+* Mule Applications are published to Exchange from Anypoint Runtime Fabric and managed in Runtime Fabric, and are not visible in the Exchange user interface.
```

Comment:
Should it be "Mule applications"?

Reply:
I checked https://docs.mulesoft.com/mule-runtime/4.1/deploying again, and I think you're right, it should be "Mule applications". Corrected. I'm also changing "API groups" while I'm in this file. I think the UI said "API Groups", but that's just title case for UI elements, not how we should refer to it in the docs.